src/main/java/at/hannibal2/skyhanni/features/mobs/SpawnTimers.kt

```diff
@@ -0,0 +1,40 @@
+package at.hannibal2.skyhanni.features.mobs
+
+import at.hannibal2.skyhanni.utils.LorenzUtils
+import at.hannibal2.skyhanni.utils.LorenzVec
+import at.hannibal2.skyhanni.utils.RenderUtils.drawDynamicText
+import at.hannibal2.skyhanni.utils.TimeUtils
+import net.minecraftforge.client.event.ClientChatReceivedEvent
+import net.minecraftforge.client.event.RenderWorldLastEvent
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+class SpawnTimers {
+    private var arachneSpawnTime = 0L
+
+    @SubscribeEvent
+    fun onRenderWorld(event: RenderWorldLastEvent) {
+        if (!LorenzUtils.inSkyBlock || arachneSpawnTime == 0L || LorenzUtils.skyBlockArea != "Arachne's Sanctuary") return;
+
+        // Arachne's Summoning Altar
+        val location = LorenzVec(-283f, 51f, -179f);
+        val countDown = arachneSpawnTime - System.currentTimeMillis()
+        if (countDown <= 0) return;
+
+        val format = TimeUtils.formatDuration(countDown - 1000, showMilliSeconds = true)
+        event.drawDynamicText(location, "§b$format", 1.5)
+    }
+
+    @SubscribeEvent
+    fun onChatReceived(event: ClientChatReceivedEvent) {

```

Comment:
Please use `LorenzChatEvent` instead
Please check if the player is on the correct island / region.
And please work with the exact chat format. Currently, the timer would start if a player writes `Something is awakening!` in the chat.